mcc/odin/odin/snapshot_list.go

```diff
@@ -1,12 +1,37 @@
 package odin
 
 import (
+	"fmt"
 	"sort"
+	"strings"
 
 	"github.com/aws/aws-sdk-go/service/rds"
 	"github.com/aws/aws-sdk-go/service/rds/rdsiface"
 )
 
+const (
+	// RFC8601 is the date/time format used by AWS.
+	RFC8601 = "2006-01-02T15:04:05-07:00"
```

Comment:
Beyond the discussion on the convenience of this formatting, the formatting in go would never refer to random dates, like `01/15/02`. If the format would be this, then the format string should be `01/02/06`.
Yes, `15` is the hour, which in their numbering, it's 3 (PM).